glean-core/ios/GleanTests/GleanTests.swift

```diff
@@ -229,9 +235,6 @@ class GleanTests: XCTestCase {
         // Set up the expectation that will be fulfilled by the stub above
         expectation = expectation(description: "Deletion Request Received")
 
-        // Reset Glean with uploadEnabled
-        Glean.shared.resetGlean(clearStores: true, uploadEnabled: true)
```

Comment:
The `setUp` function calls `resetGlean(uploadEnabled: true)` already, and it only generates a baseline ping, which the stub handler just ignores anyway.  I'll add comments to help clarify this for the next time we go wading around in this code :)

Reply:
As it turns out, we no longer need to "ignore" the baseline ping since calling `resetGlean` with upload disabled only triggers the deletion-request. The rogue one that this stub was ignoring was from the extra `resetGlean` that I removed.

Reply:
Glad I asked. The one downside with this dummy stub everywhere is that we fail to see when we accidentally trigger a startup ping that we shouldn't.